### PR TITLE
[9.0] add module 'report_mrp_bom_matrix'

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Report MRP BOM Matrix
=====================

The Matrix Bill of Materials is a report that shows connections between all
parents and all components.

This report helps to identify the products that are used in lots of
assemblies or finished products.

If a raw material/assembly is used in many parent assemblies/finished
products, it is likely that you'd want to make sure that you never run
out of it.

Usage
=====

* Go to 'Manufacturing / Products / BOM Matrix'.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
cc @lreficent

**Comment by JordiBForgeFlow** [comment]
cc @hveficent can you review?

File: report_mrp_bom_matrix/models/report_mrp_bom_matrix.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+from openerp import tools
+
+
+class ReportMrpBomMatrix(models.Model):
+    _name = 'report.mrp.bom.matrix'
+    _auto = False
```
Comment (LoisRForgeFlow) [inline report_mrp_bom_matrix/models/report_mrp_bom_matrix.py:11]: Isn't better to just use `AbstractModel`?

**Comment by JordiBForgeFlow** [comment]
@lreficent @jgrandguillaume @max3903 This one is ready to merge. Then we can start to migrate to v10 and v11

**Review by max3903 — APPROVED** [review]

**Pull request merged**